controllers/storagecluster/external_resources.go

```diff
@@ -326,6 +344,82 @@ func (obj *ocsExternalResources) ensureDeleted(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) createStorageClassClaim(instance *ocsv1.StorageCluster) error {
+
+	for _, desiredStorageClassClaim := range getDefaultStorageClassClaims(instance) {
```

Comment:
```suggestion
        for _, scc := range getDefaultStorageClassClaims(instance) {
```
Prefer using shorted names for the for loops?

Reply:
done

Reply:
I do not agree. For a reader, `scc` means nothing while `storageClassClaim` or even just `claim` has a meaning. 
We do not have any limitation on code size these days so why are we insisting on making the codebase less readable?